cogs/induct.py

```diff
@@ -115,7 +115,7 @@ async def on_member_update(self, before: discord.Member, after: discord.Member)
             "to the right & selecting \"Edit Server Profile\").",
         )
         if user_type != "member":
-            # TODO @CarrotManMatt: Remove environment variables that are only used in messages. Messages will be extracted into the external JSON file.  # noqa: FIX002
+            # TODO @MattyTheHacker: Remove environment variables that are only used in messages. Messages will be extracted into the external JSON file.  # noqa: FIX002
```

Comment:
Can we just remove this? Action tracking should be done via the github action tracker not in the code